tests/api2/test_306_chart_release.py

```diff
@@ -0,0 +1,191 @@
+    depends(request, ['release_ipfs'])
+    results = GET(f'/chart/release/id/{release_id}/')
+    assert results.status_code == 200, results.text
+    assert isinstance(results.json(), dict), results.text
+    assert results.json()['name'] == 'ipfs', results.text
+
+
+def test_10_get_chart_release_scaleable_resources():
+    results = GET('/chart/release/scaleable_resources/')
+    assert results.status_code == 200, results.text
+    assert isinstance(results.json(), dict), results.text
+
+
+@pytest.mark.dependency(name='used_ports')
+def test_11_get_chart_release_used_ports(request):
+    global port_list
+    results = GET('/chart/release/used_ports/')
+    assert results.status_code == 200, results.text
+    assert isinstance(results.json(), list), results.text
+    port_list = results.json()
+
+
+def test_12_verify_ipfs_chart_release_used_ports(request):
+    depends(request, ['release_ipfs', 'used_ports'])
+    results = GET(f'/chart/release/id/{release_id}/')
+    for port_dict in results.json()['used_ports']:
+        assert port_dict['port'] in port_list, results.text
+
+
+def test_13_get_ipfs_chart_release_upgrade_summary(request):
+    depends(request, ['release_ipfs'])
+    results = POST('/chart/release/upgrade_summary/', {'release_name': 'ipfs'})
+    assert results.status_code == 422, results.text
+    assert isinstance(results.json(), dict), results.text
+    assert 'No update is available' in results.text, results.text
+
+
+def test_14_redeploy_ipfs_chart_release(request):
+    depends(request, ['release_ipfs'])
+    results = POST('//chart/release/redeploy/', 'ipfs')
+    assert results.status_code == 200, results.text
+    assert isinstance(results.json(), int), results.text
+    job_status = wait_on_job(results.json(), 300)
+    assert job_status['state'] == 'SUCCESS', str(job_status['results'])
+
+
+def test_15_get_ipfs_chart_release_events(request):
+    depends(request, ['release_ipfs'])
+    results = POST('/chart/release/events/', 'ipfs')
+    assert results.status_code == 200, results.text
+    assert isinstance(results.json(), list), results.text
+    assert results.json()[0]['involved_object']['name'] == 'ipfs', results.text
+
+
+def test_16_get_ipfs_chart_release_pod_console_choices(request):
+    depends(request, ['release_ipfs'])
+    results = POST('/chart/release/pod_console_choices/', 'ipfs')
+    assert results.status_code == 200, results.text
+    assert isinstance(results.json(), dict), results.text
+
+
+def test_17_get_ipfs_chart_release_pod_logs_choices(request):
+    depends(request, ['release_ipfs'])
+    results = POST('/chart/release/pod_logs_choices/', 'ipfs')
+    assert results.status_code == 200, results.text
+    assert isinstance(results.json(), dict), results.text
+
+
+def test_18_set_ipfs_chart_release_scale(request):
+    depends(request, ['release_ipfs'])
+    payload = {
+        "release_name": "ipfs",
+        "scale_options": {
+            "replica_count": 0
```

Comment:
We should test if the desired pods have actually changed to the scale value set. This can be verified by `chart.release.get_instance` endpoint and from the output using the key `pod_status.desired`.